icu4c/source/tools/genrb/reslist.h

```diff
@@ -97,29 +97,30 @@ struct SRBRoot {
 public:
     // TODO: private
 
-  SResource *fRoot;  // Normally a TableResource.
-  char *fLocale;
-  int32_t fIndexLength;
-  int32_t fMaxTableLength;
-  UBool fNoFallback; /* see URES_ATT_NO_FALLBACK */
-  int8_t fStringsForm; /* default STRINGS_UTF16_V1 */
-  UBool fIsPoolBundle;
-
-  char *fKeys;
-  KeyMapEntry *fKeyMap;
-  int32_t fKeysBottom, fKeysTop;
-  int32_t fKeysCapacity;
-  int32_t fKeysCount;
-  int32_t fLocalKeyLimit; /* key offset < limit fits into URES_TABLE */
-
-  icu::UnicodeString f16BitUnits;
-  int32_t f16BitStringsLength;
-
-  const ResFile *fUsePoolBundle;
-  int32_t fPoolStringIndexLimit;
-  int32_t fPoolStringIndex16Limit;
-  int32_t fLocalStringIndexLimit;
-  SRBRoot *fWritePoolBundle;
+    SResource *fRoot;  // Normally a TableResource.
```

Comment:
Are you changing anything else here, apart from the indentation? If not, I suggest reverting these indentation changes to make the diff and the revision history easier to read.

Reply:
These changes make the indentation consistent, and I consider it necessary to make it more readable. Reverting over such a whitespace issue is unnecessary. You can always review with whitespace differences turned off. You can check out https://github.com/unicode-org/icu/pull/3766/files?w=1 to see a diff without whitespace differences.

Reply:
(the kNoPoolBundle is new here)

in the future, it would better to reformat in a separate commit, so that one can see the real changes separately